common/network-common/src/main/java/org/apache/spark/network/client/TransportClientFactory.java

```diff
@@ -86,6 +87,7 @@
   private EventLoopGroup workerGroup;
   private final PooledByteBufAllocator pooledAllocator;
   private final NettyMemoryMetrics metrics;
+  private final double fastFailTimeWindow;
```

Comment:
we don't really need this as double, just case to an int. ioRetryWaitTimeMs returns int